routes/server.go

```diff
@@ -77,6 +77,7 @@ const (
 	RoutePathGetSingleProfile                           = "/api/v0/get-single-profile"
 	RoutePathGetSingleProfilePicture                    = "/api/v0/get-single-profile-picture"
 	RoutePathGetHodlersForPublicKey                     = "/api/v0/get-hodlers-for-public-key"
+	RoutePathGetBalancesForPublicKey                    = "/api/v0/get-balances-for-public-key"
```

Comment:
```suggestion
	RoutePathGetTokenBalancesForPublicKey                    = "/api/v0/get-token-balances-for-public-key"
```
Can we make it more clear that this is for DeSo Token (FKA DAO coin) balances?

Reply:
Fixed.